model_analyzer/config/generate/generator_utils.py

```diff
@@ -116,3 +116,18 @@ def generate_doubled_list(min_value: int, max_value: int) -> List[int]:
             list.append(val)
             val *= 2
         return list
+
+    @staticmethod
+    def extract_max_tokens_from_request_parameter(request_parameter: str) -> int:
+        # format is max_tokens:<num>:int
+        _, max_tokens, _ = request_parameter.split(":")
+
+        return int(max_tokens)
+
+    @staticmethod
+    def extract_text_input_length_from_input_data(input_data: str) -> int:
+        # format is input-data-<num>.json
+        _, _, text_input_length = input_data.split("-")
+        text_input_length, _ = text_input_length.split(".")
+
```

Comment:
Remove whitespace

Reply:
I like the white space. I think it makes the return statement more readable.